CIFAR-100/train_with_distillation.py

```diff
@@ -129,9 +134,10 @@ def train_with_distill(d_net, epoch, alpha):
         b_idx = batch_idx
 
     print('Train \t Time Taken: %.2f sec' % (time.time() - epoch_start_time))
-    print('Loss: %.3f | Acc: %.3f%% (%d/%d)' % (train_loss / (b_idx + 1), 100. * correct / total, correct, total))
+    print('total_loss: %.3f | KD Loss: %.3f | CE_Loss: %.3f | Acc: %.3f%% (%d/%d)' % (total_loss / (b_idx + 1), KD_loss / (b_idx + 1), CE_loss / (b_idx + 1), 100. * correct / total, correct, total))
 
-    return train_loss / (b_idx + 1)
+
+    return total_loss / (b_idx + 1), CE_loss / (b_idx + 1), KD_loss / (b_idx + 1)
```

Comment:
```suggestion
    return total_loss / (b_idx + 1), CE_loss / (b_idx + 1), distill_loss / (b_idx + 1)
```
LGTM, 
- rename `KD_loss` -> `distill_loss`